projects/ngrx.io/content/examples/component-store-slide-toggle/src/app/slide-toggle.component.ts

```diff
@@ -78,14 +75,9 @@ export class SlideToggleComponent {
   // #enddocregion init
 
   // #docregion updater
-  onChangeEvent = this.componentStore.effect<Event>((event$) => {
```

Comment:
Is there a reason for this change?

If we update this, we should also update the docs - https://ngrx.io/guide/component-store/usage#step-2-updating-state.
Maybe that we should just keep it as is, this way we also have an example of an effect.


Thoughts?

Reply:
@timdeschryver I restored effect